modules/admin_manual/pages/maintenance/migrating_to_kiteworks.adoc

```diff
@@ -160,14 +160,23 @@ For ease of migrating external mounts, the admin should:
 
 === Kiteworks
 
-* As a major prerequisite, the Kiteworks instance *must* be running version 8.6 or higher.
+* The following major prerequisites apply:
+
+** The Kiteworks instance *must* be running version 8.6 or higher.
+** Kiteworks requires a `Data migrator` license. +
+The respective `Include feature` checkbox must be set to enabled.
 
 * You need to login into the Kiteworks appliance as role *System Admin*.
 // The kiteworks satellite service must be activated and available to the system admin user account.
 
 * The Kiteworks system must provide sufficient disk space for the data to be migrated. The ownCloud xref:migration-verification[occ migrate:to-kiteworks:verify] step will report the estimated disk space needed.
 
-* Ensure sufficient quota settings in the Kiteworks user profiles.
+* Ensure sufficient quota settings in the Kiteworks user profiles:
+
+** Users have a default quota of 2GB. +
```

Comment:
drop the 2gb default - this was the case on my machine - but who knows if this changes under what ever circumstances. furthermore this is of 0 interest - important is that we need the quota unlimited